### ISSUE TITLE
cypress-mochawesome-reporter v3.0.1 is failing to identify the package using reporter path

### ISSUE THREAD

**Issue by MoonAditya** [LironEr/cypress-mochawesome-reporter#76]
### Environment

```markdown
- OS: Windows 10 enterprise
- Node: v14.15.0
- cypress-mochawesome-reporter: 3.0.1
- cypress: 8.7.0
```

### What happened?

The issue is related to the
**cypress-mochawesome-reporter**: { "version": "**3.0.1**"}
this version has a strict check for the reporter name in the **config.js** file, see the code here

`if (config.reporter === 'cypress-mochawesome-reporter') {
    return opts;
  }`
  
It is a failing and logging a message that "cypress-mochawesome-reporter is not an active reporter" since we are providing the path of the package in our node module.
I am only using this reporter.
However, due to our project setup, we need the flexibility to pass the reporter value as a path along with the package name. 
such as,

`"reporter": "../../node_modules/cypress-mochawesome-reporter",`

### Config file

```json
see the cypress.json file set up here

{
    "fileServerFolder": ".",
    "fixturesFolder": "./src/fixtures",
    "integrationFolder": "./src/integration",
    "modifyObstructiveCode": false,
    "pluginsFile": "./src/plugins/index",
    "supportFile": "./src/support/index.ts",
    "video": false,
    "videosFolder": "../../cypress-reports/videos",
    "screenshotsFolder": "../../cypress-reports/screenshots",
    "chromeWebSecurity": false,
    "reporter": "../../node_modules/cypress-mochawesome-reporter",
    "reporterOptions": {
        "reportDir": "../../cypress-reports",
        "embeddedScreenshots": true,
        "overwrite": false,
        "charts": true,
        "showSkipped": true,
        "saveJson": true
    }
}
```

### Relevant log output

```shell
cypress-mochawesome-reporter is not the active reporter, ignore before run hook.

cypress-mochawesome-reporter is not the active reporter, ignore after run hook.
```

### Anything else?

It should not be failing if we pass the path of the package as reporter.

**Comment by jinisha** [comment]
I am having a similar problem.

Env:- 
- OS: Windows 10 enterprise
- Cypress:        9.5.1                                                                         
- Browser:        Electron 94 (headless)                                                        
- Node Version:   v16.13.1
- cypress-mochawesome-reporter: 3.0.1
- cypress: 8.7.0

cypress.json setup

"reporter": "../../../node_modules/cypress-multi-reporters",
  "reporterOptions": {
    "configFile": "reporter-config.json"
  },

reporter-config.json
{
  "reporterEnabled": "cypress-mochawesome-reporter",
  "cypressMochawesomeReporterReporterOptions": {
    "reportDir": "cypress/reports",
    "embeddedScreenshots": true,
    "charts": true,
    "inlineAssets": true
  }
}

When I start running the tests, it throws **cypress-mochawesome-reporter is not the active reporter, ignore before run hook** error

**Comment by LironEr** [comment]
Should be fixed in [v3.1.0](https://github.com/LironEr/cypress-mochawesome-reporter/releases/tag/v3.1.0).
